phpBB/phpbb/storage/storage.php

```diff
@@ -133,11 +140,64 @@ public function rename($path_orig, $path_dest)
 	 * @param string	$path_orig	The original filename
 	 * @param string	$path_dest	The target filename
 	 *
-	 * @throws \phpbb\storage\exception\exception	When target exists
-	 * 												When the file cannot be copied
+	 * @throws \phpbb\storage\exception\exception		When target exists
+	 * 													When the file cannot be copied
+	 * @throws \phpbb\storage\exception\not_implemented	When the adapter doesnt implement the method
```

Comment:
Ok, i can remove not_implemented exception directly if you want, and they just throws a normal exception like if a generic error ocurred